vng_api_common/pagination.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework.pagination import PageNumberPagination
+
+
+class DynamicPageSizeMixin:
+    page_size = 100
+    page_size_query_param = "pageSize"
+    max_page_size = 500
```

Comment:
```suggestion
```

Reply:
I don't quite understand this suggestion. Do you want the mixin to be removed? 
In this way It won't be used for custom pagination classes (like in Open Zaak)

Reply:
I had not thought about re-using the mixin for other projects so you can ignore the suggestion :+1: